packages/changed-elements-react/src/api/VersionCompareManager.ts

```diff
@@ -408,15 +410,15 @@ export class VersionCompareManager {
       this.loadingProgressEvent.raiseEvent(
         IModelApp.localization.getLocalizedString("VersionCompare:versionCompare.msg_openingTarget"),
       );
-
+      this._isComparisonInitializing = true;
       // Open the target version IModel
       const changesetId = targetVersion.changesetId;
       this._targetIModel = await CheckpointConnection.openRemote(
         this._currentIModel.iTwinId,
         this._currentIModel.iModelId,
         IModelVersion.asOfChangeSet(changesetId),
       );
-
+      this._isComparisonInitializing = false;
```

Comment:
I don't think this is the appropriate place to do this. Maybe look at what we had before? It was done at the end of the loading, because the awaits below can take a long time...
[see your last change](https://github.com/iTwin/changed-elements-react/commit/4d48ceb9e54e0f1438e64627b62b3f81aa75f498#diff-4e78b3346446ea4f514f597589225dea4c5e24a8b4f7667de76a963e1845abd8L368)